cargo-dist/tests/snapshots/error_manifest.snap

```diff
@@ -6,7 +6,10 @@ stdout:
 {"diagnostic": {"message": "This workspace doesn't have anything for cargo-dist to Release!","severity": "error","causes": [],"labels": [],"related": []}}
 
 stderr:
-analyzing workspace:
+  cargo-dist
+    [bin] cargo-dist
+  cargo-dist-schema (no binaries)
+
```

Comment:
these lines are still getting double-printed. it's also an interesting point that we're not forwarding --tag in this... probably right on balance